diff --git a/node_modules/node-webcrypto-ossl/index.d.ts b/node_modules/node-webcrypto-ossl/index.d.ts
index 7af040a..e98e956 100644
--- a/node_modules/node-webcrypto-ossl/index.d.ts
+++ b/node_modules/node-webcrypto-ossl/index.d.ts
@@ -174,7 +174,7 @@ declare namespace NodeWebcryptoOpenSSL {
     }
 }
 
-declare const NodeWebCrypto: typeof NodeWebcryptoOpenSSL.WebCrypto;
+declare const NodeWebCrypto: { Crypto: typeof NodeWebcryptoOpenSSL.WebCrypto };
 
 declare module "node-webcrypto-ossl" {
     export = NodeWebCrypto;
